Build/index.md

```diff
@@ -14,15 +14,15 @@ Both [NJOY21](https://github.com/njoy) and [NJOY2016](https://github.com/njoy) u
 # Download the source code
 git clone https://github.com/njoy/NJOY21.git
 
-# Get the desired version of NJOY21 (1.0.1 in this example)
+# Get the desired version of NJOY21 (1.1.0 in this example)
 cd NJOY21
-wget https://raw.githubusercontent.com/njoy/signatures/master/NJOY21/1.0.1-NJOY21.json
-./metaconfigure/fetch_subprojects.py 1.0.1-NJOY21.json
+wget https://raw.githubusercontent.com/njoy/signatures/master/NJOY21/1.1.0-NJOY21.json
+./metaconfigure/fetch_subprojects.py 1.1.0-NJOY21.json
 
 # Configure the build process
 mkdir bin
 cd bin
-cmake -D fetched_subprojects=true -D CMAKE_BUILD_TYPE=release ../
```

Comment:
Did we set a default build type?

Reply:
We did not.  Without specifying, `CMAKE_BUILD_TYPE` defaults to a blank string.  And if I understand CMake correctly (big if...), neither the release nor the debug flags are added to the compilation command.

Reply:
Yes, that's what happens. I do that all the time.

You can actually look at the cache of cmake using ccmake . in the build/bin directory. The GUI will tell you what the value of each variable is.